gpiozero/output_devices.py

```diff
@@ -180,6 +184,10 @@ def blink(self, on_time=1, off_time=1, n=None, background=True):
             self._blink_thread.join()
             self._blink_thread = None
 
+    def _stop_source(self):
+        if self.source is not None:
```

Comment:
I guess this if statement is redundant. If it's None already, setting it to None will do no harm.